torch/fx/graph_module.py

```diff
@@ -106,7 +106,9 @@ def _format_import_block(globals: Dict[str, Any], importer: Importer):
     import_strs: Set[str] = set()
     for name, obj in globals.items():
         import_strs.add(_format_import_statement(name, obj, importer))
-    return "\n".join(import_strs)
+    # Sort the imports so we have a stable import block that allows us to
+    # hash the graph module and get a consistent key for use in a cache.
+    return "\n".join(sorted(import_strs))
```

Comment:
@eellison I thought it was just the behavior of Python's set. I don't think set insertion preserves anything.

Reply:
where as we using set ? maybe we should have a linter rule in torch/_inductor to never use set(), and always use an ordered set() variant

Reply:
Just a few lines above on line 106: `import_strs: Set[str] = set()`
Sure; I can change the usage here and I'm pretty certain it will work fine. But what ordered set variant? Do you know of one? I was surprised to find that collections lib, for example, doesn't have one. Googling and ChatGPT'ing point to rando third-party libs that I don't think we're currently using anywhere this codebase. I could do it with a dict?